data_processing.py

```
@@ -1,11 +1,10 @@
-        item = data[i]
-        if isinstance(item, int) and item > 0:
-            transformed_item = item * 2
-            result.append(transformed_item)
-        else:
-            result.append(0)
-    return result
\ No newline at end of file
+def process_data(data): 
+     if data is None: 
+         raise ValueError('Input data cannot be None')  
+     try: iter(data)  # Check if data is iterable 
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)